### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by sakkshm26**
Fixes: #1982

### CODE REVIEW

**Review by hwdef — COMMENTED** [review]
/lgtm

**Comment by hwdef** [comment]
please fix the DCO

**Review by jasonliu747 — APPROVED** [review]
/lgtm

File: contribute.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Please make sure to read and observe our [Code of Conduct](./code_of_conduct.md)
 ## Community Expectations
 
 Volcano is a community project driven by its community which strives to promote a healthy, friendly and productive environment.
-The goal of the community is to develop a volcano system which is useful for running high performance workloads such as AI, ML, Deep Learning Application on top of Kubernetes. To build a such volcano system at such scale requires the support of a community with similar aspirations.
+The goal of the community is to develop a volcano system which is useful for running high performance workloads such as AI, ML, Deep Learning Application on top of Kubernetes. To build such a volcano system at such scale requires the support of a community with similar aspirations.
```
Comment (jasonliu747) [inline contribute.md:27]: --> To build a volcano system at such scale

File: community-membership.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ Has deep understanding of Volcano and related domain and facilitates major agree
 - Participate & Drive design/development of multiple features
 
 
-**Note :** These roles are applicable only for Volcano github organization and repositories. Currently Volcano doesn't have a formal process for review and acceptance into these roles. We will come-up with a process soon.
+**Note :** These roles are applicable only for Volcano github organization and repositories. Currently, Volcano doesn't have a formal process for review and acceptance into these roles. We will come-up with a process soon.
```
Comment (jasonliu747) [inline community-membership.md:116]: --> We will come up with

**Comment by Thor-wl** [comment]
Really appreciate some work about improving the docs. We are discussing about the posibility of setting Doc SIG for Volcano. Welcome to join the SIG if it is put into practice.

**Comment by Thor-wl** [comment]
Pls fix the DCO by execute `git commit --signoff -m "xxx"` to add the signature.

**Pull request merged**